server/common/HatoholArmPluginInterface.cc

```diff
@@ -157,6 +157,10 @@ struct HatoholArmPluginInterface::Impl {
 		connected = false;
 		hapi->onSessionChanged(NULL);
 	}
+	
+	bool connectstat(void) {
```

Comment:
The name is unreadable.
Some people may think it makes a connection because it begin with a verb: connect.